cfgov/v1/models/browse_page.py

```diff
@@ -72,11 +76,31 @@ class BrowsePage(CFGOVPage):
 
     objects = PageManager()
 
+    def get_mortgage_meta(self):
```

Comment:
Can this function live on the `MortgageChartBlock` so as not to clutter up `BrowsePage`?